profiles/use.desc

```diff
@@ -0,0 +1 @@
+lv2 - Add support for Ladspa V2
```

Comment:
Nobody calls it Ladspa V2 (even though that's where the name comes from), and this doesn't say much about what the flag actually does. Should be something like "Add support for LV2 plug-ins"